docs/source/tutorials/faq.ipynb

```diff
@@ -611,6 +611,53 @@
     "cl.fit(X, y, find_label_issues_kwargs={\"coarse_search_range\": [0.05, 0.1], \"fine_search_size\": 2})"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How do I generate cluster IDs and pass them to find_issues for the Underperforming Group Issue Manager?"
```

Comment:
```suggestion
    "### How do I specify pre-computed data slices/clusters when detecting the Underperforming Group Issue?"
```

Reply:
we should really minimize mention of IssueManagers in anything user facing. Users should not have to know what issue managers are, they are supposed to do everytthing via Datalab directly